conf/evolutions/default/11.sql

```diff
@@ -0,0 +1,7 @@
+# --- !Ups
+
+ALTER TABLE "MenuPerDay" ADD COLUMN "location" TEXT NOT NULL;
```

Comment:
You can't just add a non null column to an existing database. Remember that the DB in production already has data.
Or you define a default value or you create this in a few more steps (new column nullable, update column with data 'Rotterdam' and then modify it to be nullable).